paper-checkbox.html

```diff
@@ -138,18 +143,23 @@
 
       #checkmark {
         position: absolute;
-        width: 5px;
-        height: 10px;
+        width: 36%;
+        height: 70%;
         border-style: solid;
         border-top: none;
         border-left: none;
-        border-right-width: 2px;
-        border-bottom-width: 2px;
+        border-right-width: calc(2/15 * var(--calculated-paper-checkbox-size));
+        border-bottom-width: calc(2/15 * var(--calculated-paper-checkbox-size));
         border-color: var(--paper-checkbox-checkmark-color, white);
         transform-origin: 97% 86%;
         -webkit-transform-origin: 97% 86%;
       }
 
+      :host-context([dir="rtl"]) #checkmark {
+        transform-origin: 50% 14%;
+        -webkit-transform-origin: 50% 14%;
+      }
```

Comment:
Does this change also fix an existing RTL bug? If so, you should totally take credit and close the issue ;)